components/widgets/api/src/main/resources/ApplicationResources.properties

```diff
@@ -13,12 +13,13 @@ phenotips.imageDisplayer.done=Done
 phenotips.imageDisplayer.noFiles=No files available
 
 phenotips.tableMacros.delete=Delete
-phenotips.tableMacros.reject=Reject
 phenotips.tableMacros.newEntry=New entry
+phenotips.tableMacros.newVariantEntry=Add variant
 phenotips.tableMacros.noObjects=None specified
 phenotips.tableMacros.rowDeleteConfirmation=Are you sure you want to delete this row?
 phenotips.tableMacros.listNotFound=Cannot find the list to update
-phenotips.tableMacros.rejectConfirmation=Are you sure you want to reject this row?
+phenotips.tableMacros.typeNotFound=Cannot add data: type not found
```

Comment:
That's exactly what we have in [TabelarDataMacros](https://github.com/phenotips/phenotips/blob/master/components/widgets/ui/src/main/resources/PhenoTips/TabelarDataMacros.xml) line 238. I'll change to "Cannot add data: variant object not found"